libs/wire-api/src/Wire/API/Error/Brig.hs

```diff
@@ -194,3 +195,5 @@ type instance
 type instance MapError 'InvalidPwResetKey = 'StaticError 400 "invalid-key" "Invalid email or mobile number for password reset."
 
 type instance MapError 'DuplicatePwResetCode = 'StaticError 409 "code-exists" "A password reset is already in progress."
+
+type instance MapError 'InvalidPwResetCode = 'StaticError 400 "invalid-code" "Invalid password reset code."
```

Comment:
This is currently only used for swagger, I think we should also use this to create the actually error being thrown, you can use `errorToWai`. There are examples in the `Brig.API.Error` module.